Automattic/woocommerce-connect-client
# Hook up edit package link (redux)

Fixes #240.

To test:
* Click a package's name
* Verify that the modal pops up, says "Edit package" at the top, button says "apply changes"
* Change the package name
* Click "apply changes"
* Verify package name has changed in list
* Click "add package" (under packages list)
* Verify form is empty, says "Add package" at top, "package type" dropdown is present, button says "add package"
* Select a USPS preset from the "package type" dropdown
* Verify that form values are populated with data
* Hit ESC or click off the modal to dismiss it
* Click "add package" (under packages list)
* Verify all previous values are present
* Click "add package" (on modal)
* Verify package had been added to list
